content/api/module-methods.md

```diff
@@ -53,7 +53,11 @@ export default {
 
 ### `import()`
 
-Dynamically load modules.
+`import('path/to/module') -> Promise`
```

Comment:
Can we remove the `-> Promise` part, as it looks like something that the user should actually type in their code.

Maybe something like:
`import('path/to/module') // returns a Promise`